parm/config/config.vrfy

```diff
@@ -20,8 +20,8 @@ export VSDB_STEP1="YES"           # Populate VSDB database
 export VSDB_STEP2="NO"
 export VRFYG2OBS="YES"            # Grid to observations, see note below if turning ON
 export VRFYPRCP="YES"             # Precip threat scores
-export VRFYRAD="YES"              # Radiance data assimilation monitoring
-export VRFYOZN="YES"              # Ozone data assimilation monitoring
+export VRFYRAD="NO"              # Radiance data assimilation monitoring
+export VRFYOZN="NO"              # Ozone data assimilation monitoring
```

Comment:
Curious, why changed to "NO"?

Reply:
This was left in by mistake - at one point I had to turn these off due to linking failures caused by cross-hash incompatibilities. Will correct.